post_installation_configuration/aws-compute-edge-tasks-wavelength.adoc

```diff
@@ -0,0 +1,139 @@
+:_mod-docs-content-type: ASSEMBLY
+include::_attributes/common-attributes.adoc[]
+
+:context: aws-compute-edge-tasks-wavelength
+
+// Context Variables for edge zones
+:aws-service-name-full: AWS Wavelength
+:aws-service-name-short: Wavelength Zones
+:aws-service-name-single: Wavelength Zone
+:zone-type: wavelength-zone
+
+[id="aws-compute-edge-tasks-wavelength"]
+= AWS Wavelength tasks
+
+// ATTENTION: All the content from here must be shared between books:
+// aws-compute-edge-tasks
+// installing-aws-wavelength
+
+ifeval::["{zone-type}" == "{aws-zone-type-local}"]
+:aws-localzones:
+:aws-doc-zone: https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/using-regions-availability-zones.html#concepts-local-zones
+endif::[]
+ifeval::["{zone-type}" == "{aws-zone-type-wavelength}"]
+:aws-wavelength:
+:aws-doc-zone: https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/using-regions-availability-zones.html#concepts-wavelength-zones
+endif::[]
+
+toc::[]
+
+After installing {product-title} on Amazon Web Services (AWS), you can further configure {aws-service-name-full} and an edge
+compute pool, so that you can expand and customize your cluster to meet your needs.
+
+// Extend existing clusters to use {aws-service-name-full}
+include::modules/post-install-edge-aws-extend-cluster.adoc[leveloffset=+1]
+
+// Additional Permissions for {aws-service-name-full}
+include::modules/aws-edge-zones-permissions.adoc
+
+[role="_additional-resources"]
+.Additional resources
+
+* For more information about {aws-service-name-full}, the supported instances types, and services,
+ifdef::aws-localzones[]
+  see link:https://aws.amazon.com/about-aws/global-infrastructure/localzones/features/[AWS Local Zones features]
+endif::aws-localzones[]
+ifdef::aws-wavelength[]
+  see link:https://aws.amazon.com/wavelength/features[{aws-service-name-full} features]
+endif::aws-wavelength[]
+  in the AWS documentation.
+
+// About the Edge Compute Pool
+include::modules/edge-machine-pools-aws-local-zones.adoc[leveloffset=+2]
+
+[id="post-install-extend-existing-to-local-zones-mtu"]
+=== Changing the Cluster Network MTU to support {aws-service-name-full} subnets
+
+You might need to change the maximum transmission unit (MTU) value for the cluster
+network so that your cluster infrastructure can support communication between nodes
+in the Region and in the {aws-service-name-single}.
+
+// About the cluster MTU
+include::modules/nw-cluster-mtu-change-about.adoc[leveloffset=+3]
+
+// Changing the cluster MTU
```

Comment:
Hi @mtulio . The preview on my PR has slightly different steps for Changing the cluster MTU section.  https://69537--ocpdocs-pr.netlify.app/openshift-enterprise/latest/post_installation_configuration/aws-compute-edge-tasks-local-zones#nw-cluster-mtu-change_aws-compute-edge-tasks-local-zones

I cannot determine why this is, but should the steps in this section match your preview?